src/mixins/filing-template-mixin.ts

```diff
@@ -321,9 +322,18 @@ export default class FilingTemplateMixin extends DateMixin {
       filing.restoration.expiry = this.getRestoration.expiry
     }
 
-    // Apply NR / business name / business type change to filing
-    if (this.getNameRequestNumber || this.hasBusinessNameChanged || this.hasBusinessTypeChanged) {
-      filing.restoration.nameRequest = this.getNameRequest
+    // Set Name Request object for a full restoration only
+    if (this.isLimitedRestorationToFull) {
```

Comment:
so do full restoration still provide the NR block with this change?

Reply:
I don't know. Those are done in Create UI.

Reply:
oh, this is edit ui.  nvm

Reply:
![image](https://user-images.githubusercontent.com/10002889/232169890-7d7f9bc3-d801-443d-83d5-d323f67097d7.png)

Reply:
For a Full Restoration, these is a default Name Request object, with a default property (line 423).

Then, depending on how the business name is "corrected", more properties may be added (lines 439-452).